R/special.R

```diff
@@ -0,0 +1,47 @@
+#' Special dispatch types
+#'
+#' * Use `missing_class` when you the user has not supplied an argument
```

Comment:
```suggestion
#' * Use `missing_class` when the user has not supplied an argument
```

I think you didn't want this